rwasm/wasmrunnable.go

```diff
@@ -80,7 +80,8 @@ func (w *Runner) Run(job rt.Job, ctx *rt.Ctx) (interface{}, error) {
 		// ident is a random identifier for this job run that allows for "easy" FFI function calls in both directions
 		if _, wasmErr := wasmRun(inPointer, len(jobBytes), ident); wasmErr != nil {
 			runErr = errors.Wrap(wasmErr, "failed to wasmRun")
-			return
```

Comment:
Pointing out here that the behaviour of the Wasm runner has changed; wasmRun error is no longer returned outright in case a more helpful error was returned in the select down below. If it wasn't, we fall through and return the wasmErr as normal.